### DISCUSSION TITLE
Customize the contents of libraries

### DISCUSSION THREAD

**Discussion by ajpanton** [jellyfin/jellyfin#12468] [General]
I'd like to have more control over what is included in a library. For example to exclude files/videos with certain criteria.

Example: I want to separate full-length movies and shorts. A movie library could exclude videos with length less than say 20 minutes. A shorts library could exclude videos with length more than or equal to 20 minutes.

I guess something could be done by re-organizing the whole folder structure, but that's not always possible or desireable if Jellyfin is not the only thing accessing them.

**Comment by TheMelmacian** [comment]
You could try these alternatives to achieve something similar:

- collections: Put all shorts in one collection and the full-length movies in another one. Then you have them grouped together.
- tags: Define tags for shorts and movies and add them accordingly. Then you can easily filter by these tags.
